spec/controllers/api/v0/user_roles_controller_spec.rb

```diff
@@ -125,6 +125,15 @@
         expect(response_json["error"]).to eq "The user has upcoming competitions: #{upcoming_comps_for_user.join(', ')}. Before banning the user, make sure their registrations are deleted."
       end
 
+      it 'can ban a user if the user\'s upcoming competitions are after end date' do
```

Comment:
You can make this double quotes, then you don't have to mask out the apostrophe in `don't` 
```suggestion
      it "can ban a user if the user's upcoming competitions are after end date" do
```

Reply:
Done